src/modules/assessments/components/household/HouseholdAssessments.tsx

```diff
@@ -269,8 +269,8 @@ const HouseholdAssessments: React.FC<Props> = ({
           top: STICKY_BAR_HEIGHT + CONTEXT_HEADER_HEIGHT,
           backgroundColor: 'white',
           px: { sm: 3, lg: 5 },
-          ml: { xs: -2, sm: -3, lg: -4 },
-          mt: { xs: -1, sm: -2 },
+          ml: { xs: -1, sm: -3, lg: -4 },
```

Comment:
This nav bar for household assessments still needs more work to be mobile friendly, but here's how it looks now:
<img width="361" alt="Screenshot 2024-03-08 at 2 34 41 PM" src="https://github.com/greenriver/hmis-frontend/assets/16002775/32584a99-dd3c-4744-b347-d4d14584181d">